### PR TITLE
commit BrooksJava

### PR DESCRIPTION
**Pull request by BrooksJava**
我是你爸爸

### CODE REVIEW

File: hello.txt [hunk 1]
```diff
@@ -3,7 +3,4 @@ hello git! --modified secend
 hello git!
 hello git!  location at branch master 
 hello git! location at hot-fix
-hello  ll
-add one line by BrooksGit
-add one line by BrooksGit
-add one line by BrooksGit
+I am your father! modified by BrooksJava
```
Comment (GeJavaGoHuai) [inline hello.txt:6]: fuck you man!

**Review by GeJavaGoHuai — COMMENTED** [review]
ugly boy

**Comment by GeJavaGoHuai** [comment]
ugly girl?

**Comment by BrooksJava** [comment]
擅吃鱼

**Pull request merged**